bubuku/id_generator.py

```diff
@@ -51,9 +51,15 @@ def _create_rfc1918_address_hash(ip: str) -> (str, str):
 class BrokerIDByIp(BrokerIdGenerator):
     def __init__(self, zk: BukuExhibitor, ip: str, kafka_props: KafkaProperties):
         self.zk = zk
-        self.broker_id, max_id = _create_rfc1918_address_hash(ip)
+        self.broker_id = _read_broker_id_from_meta_properties(kafka_props)
+        max_id = str(256 * 256 * 256 * 4 + 1)
```

Comment:
Why not put this line in the `else:` block starting on line 59? Here, it can get re-written on line 57 if `self.broker_id` is `None`